ramls/loan-policy.json

```diff
@@ -85,6 +85,69 @@
         }
       }
     },
+    "requestManagement": {
+      "type": "object",
```

Comment:
Each of the object properties needs to include `"additionalProperties": false` to disallow additional properties within it

Reply:
added

Reply:
It appears they were added to all of the object properties. Sorry, that was likely my poor description. If we add them to the existing objects, that would be a breaking change. I meant only the new object properties added during this change. Apologies :-(

Reply:
removed for existing objects

Reply:
Thanks